modules/common/util/util.h

```diff
@@ -268,6 +270,29 @@ IsFloatEqual(T x, T y, int ulp = 2) {
 }  // namespace common
 }  // namespace apollo
 
+template<typename T>
+class FunctionInfo {
+ public:
+  typedef int(T::*Function)();
+  Function function_;
+  std::string fun_name_;
+};
+
+template<typename T, size_t count>
+bool ExcuteAllFunctions(T *obj, FunctionInfo<T> fun_list[]) {
+  for (size_t i = 0; i< count ; i++) {
+    if ((obj->*(fun_list[i].function_))() != apollo::cyber::SUCC) {
+      AERROR << fun_list[i].fun_name_ <<" failed.";
+      return false;
+    }
+  }
+  return true;
+}
+
```

Comment:
Please don't introduce template is possible if that only would be used for one thing.

Reply:
The template is used at least in two places.